docs/source/basics/building_a_pipeline.md

```diff
@@ -0,0 +1,257 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Building a Pipeline
+> **Prerequisites**
+> The following examples assume that the example dataset has been fetched, and that a `.tmp` directory has been created at the root of the Morpheus repo. `.tmp` has an entry in `.gitignore` and any files in that directory will be automatically ignored by git. From the root of the Morpheus repo run:
+>```bash
+>./scripts/fetch_data.py fetch examples
+>mkdir -p .tmp
+>```
+
+To build a pipeline via the CLI, users must first specify the type of pipeline, a source object, followed by a sequential list of stages. For each stage, options can be specified to configure the particular stage. Since stages are listed sequentially the output of one stage becomes the input to the next. Unless heavily customized, pipelines will start with either:
+
+```bash
+# For NLP Pipelines
+morpheus run pipeline-nlp ...
+# For FIL Pipelines
+morpheus run pipeline-fil ...
+```
+
+While each stage will have configuration options, there are options that apply to the pipeline as a whole as well. Check
+``morpheus run --help``, ``morpheus run pipeline-nlp --help`` and ``morpheus run pipeline-fil --help`` for these global
+Pipeline options.
+
+## Source Stages
+
+All pipelines configured with the CLI need to start with a source object. Two commonly used source stages included with Morpheus are:
+
+* `from-file`
+  - Reads from a local file into the Pipeline
+  - Supports JSON lines format
+  - All lines are read at the start and queued into the pipeline at one time. Useful for performance testing.
+  - Refer to `morpheus.pipeline.input.from_file.FileSourceStage` for more information
+* `from-kafka`
+  - Pulls messages from a Kafka cluster into the Pipeline
+  - Kafka cluster can be remote or local
+  - Refer to `morpheus.pipeline.input.from_kafka.KafkaSourceStage` for more information
```

Comment:
Could you replace `morpheus.pipeline.input.from_kafka.KafkaSourceStage` with `morpheus.stages.input.from_kafka.KafkaSourceStage`?